packages/styleguide/src/style/theme.js

```diff
@@ -4,7 +4,7 @@ import { rem } from './utils';
 
 export const sizes = {
   headerHeight: '96px',
-  sidebarWidth: '248px',
```

Comment:
Informacne, toto sa preco menilo?

Reply:
ked je menu moc dlhe tak sa v nom zobrazi scroll bar. Lenze ked ho vtedy zavres tak ono odbočí dolava o 248px čo je malo lebo stale vidno vytrčaju scrollbar... tak som to len napevno zvysil a uz ho nevidno